tests/PHPStan/Command/ErrorFormatter/TableErrorFormatterTest.php

```diff
@@ -13,6 +13,9 @@
 use function sprintf;
 use const PHP_VERSION_ID;
 
+/**
+ * @runTestsInSeparateProcesses
+ */
 class TableErrorFormatterTest extends ErrorFormatterTestCase
```

Comment:
I just think that this test was influenced by some other test that set an env variable.

This diff is a workaround, not a solution to the problem.

Reply:
yeah, I am still in the process of finding a proper solution. the tests don't fail locally, so I push and look at CI results